benefits/friday_lunch.md

```diff
@@ -1,17 +1,11 @@
 # Friday Lunch
 
-Every Friday we pay for a team lunch.  This weekly ritual provides an opportunity for the team to socialise and share their successes and challenges from the week that's passed. It's important that we continue to set aside the time for this, especially when we have teams working at customers offices regularly.
+Every Friday we pay for 5 people from each region to have a team lunch. This weekly ritual provides an opportunity for the team to socialise and share their successes and challenges from the week that's passed. It's important that we continue to set aside the time for this, especially when we're all working remotely.
 
-Few rules:
+How it works:
+- the People team will organise lunches, inviting people from each region: you can only expense your lunch if invited to do so by them.
```

Comment:
Looks good thanks for making these changes. Just an inconsistent capitalisation here? Apart from that happy to approve and merge 👍